HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
 
-# Your code here
+import matplotlib.pyplot as plt
+import numpy as np
+import random
+
+X = range(2000)
+Y = range(2000)
+random.shuffle(X) 
+random.shuffle(Y)
+
```

Comment:
It is interesting that you improve the partition strategy using random.shuffle, however, in order to be more targeted and efficient, customise partition strategy according to pixels distribution on the graph using spark "partitioner" could be an optimised solution.